src/core/server/saved_objects/service/lib/aggregations/validation_utils.ts

```diff
@@ -24,15 +24,16 @@ export const isRootLevelAttribute = (
   allowedTypes: string[]
 ): boolean => {
   const splits = attributePath.split('.');
-  if (splits.length !== 2) {
+  if (splits.length <= 1) {
     return false;
   }
 
-  const [type, fieldName] = splits;
-  if (allowedTypes.includes(fieldName)) {
+  const [type, ...fieldName] = splits;
+  if (allowedTypes.includes(fieldName.join('.'))) {
     return false;
   }
```

Comment:
Does that really protect against `alert.alert.group.[...]` ? 

I think we need something like

```ts
const [type, firstPath, ...otherPaths] = splits;
if (allowedTypes.includes(firstPath)) {
```